release.nix

```diff
@@ -125,21 +134,26 @@ let
 
   jobs = {
     inherit dockerImageArtifact;
-    native =
+    linux =
       let filteredBuilds = mapAttrsRecursiveCond (a: !(isList a)) (path: value:
         if (any (p: take (length p) path == p) onlyBuildOnDefaultSystem) then filter (s: !(elem s nonDefaultBuildSystems)) value else value)
         (packagePlatforms project);
       in (mapTestOn (__trace (__toJSON filteredBuilds) filteredBuilds));
+    darwin =
```

Comment:
I would prefer to call it `macos` instead of `darwin` where possible - less jargon.